contracts/Authorizable.sol

```diff
@@ -4,18 +4,23 @@ pragma solidity >=0.4.22 <0.9.0;
 import "@openzeppelin/contracts/access/Ownable.sol";
 
 contract Authorizable is Ownable {
-    address public trustee;
+    address[] public trustees;
 
-    constructor() {
-        trustee = address(0x0);
-    }
+    constructor() {}
 
     modifier onlyAuthorized() {
-        require(msg.sender == trustee || msg.sender == owner());
+        bool isAuthorized = false;
+        for (uint256 i = 0; i < trustees.length; i++) {
```

Comment:
I prefer using map for `trustees` so that we do not need to do for loop. Otherwise, the cost of execution would be varied by the index of a trustee.